content/systems/filecoin_mining/storage_mining/_index.md

```diff
@@ -1,65 +1,90 @@
 ---
 title: Storage Miner
 bookCollapseSection: true
-weight: 1
+weight: 2
 dashboardWeight: 2
-dashboardState: incorrect
+dashboardState: wip
 dashboardAudit: wip
 dashboardTests: 0
 ---
 
 # Storage Miner
 
-## Filecoin Storage Mining Subsystem
+## Storage Mining Subsystem
 
 The Filecoin Storage Mining Subsystem ensures a storage miner can effectively commit storage to the Filecoin protocol in order to both:
 
 - Participate in the Filecoin [Storage Market](storage_market) by taking on client data and participating in storage deals.
-- Participate in Filecoin [Storage Power Consensus](storage_power_consensus), verifying and generating blocks to grow the Filecoin blockchain and earning block rewards and fees for doing so.
+- Participate in Filecoin [Storage Power Consensus](storage_power_consensus) by verifying and generating blocks to grow the Filecoin blockchain and earning block rewards and fees for doing so.
 
 The above involves a number of steps to putting on and maintaining online storage, such as:
 
-- Committing new storage (see Sealing and PoRep)
+- Committing new storage (see [Sector](sector), [Sector Sealing](sector#sealing) and [PoRep](sdr))
 - Continuously proving storage (see [Election PoSt](election_post))
```

Comment:
Election PoSt is not used in the protocol anymore. We currently have only WinningPoSt and WindowPoSt.
So I recommend to change

    Continuously proving storage (see Election PoSt)

in
" Continuously proving storage (see Winning PoSt and Window Post (window_post))"

hopefully the links there are correct!